### DISCUSSION TITLE
https://chat.openai.com/g/g-b2BSBtDWm

### DISCUSSION THREAD

**Discussion by giscus** [lencx/gpthub#37] [Comments]
# https://chat.openai.com/g/g-b2BSBtDWm

Discover the Best in Custom GPT at OpenAI's GPT Store – Your Adventure Begins Here!

https://gpthub.nofwl.com/

**Comment by AbuWaleedsy2** [comment]
هل يمكنك استخراج البيانات من بي دي اف الى اكسل
